app/sidekiq/central_mail/submit_form4142_job.rb

```diff
@@ -131,20 +136,39 @@ def lighthouse_service
       @lighthouse_service ||= BenefitsIntakeService::Service.new(with_upload_location: true)
     end
 
-    def upload_to_lighthouse
-      Rails.logger.info(
-        'Successful Form4142 Submission to Lighthouse',
-        { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
-      )
-
-      payload = {
-        upload_url: lighthouse_service.location,
+    def payload_hash(lighthouse_service_location)
+      {
+        upload_url: lighthouse_service_location,
         file: { file: @pdf_path, file_name: @pdf_path.split('/').last },
         metadata: generate_metadata.to_json,
         attachments: []
       }
+    end
 
+    def upload_to_lighthouse
+      log_info = { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
+
+      Rails.logger.info(
+        'Successful Form4142 Upload Intake UUID aquired from Lighthouse',
+        log_info
+      )
+
+      payload = payload_hash(lighthouse_service.location)
       lighthouse_service.upload_doc(**payload)
+
+      if Flipper.enabled?(POLLING_FLIPPER_KEY)
+        form526_submission = Form526Submission.find(@submission_id)
+        form_submission    = FormSubmission.create(
+          form_type: FORM4142_FORMSUBMISSION_TYPE, # form526_form4142
+          benefits_intake_uuid: lighthouse_service.uuid,
+          form_data: '{}', # we have this already in the Form526Submission.form['form526']['form526']['form4142']
+          user_account: form526_submission.user_account,
+          saved_claim: form526_submission.saved_claim
+        )
+        FormSubmissionAttempt.create(form_submission:)
+        log_info[:form_submission_id] = form_submission.id
+      end
+      Rails.logger.info('Successful Form4142 Submission to Lighthouse', log_info)
```

Comment:
I think no, because even if the flipper is off, I still want it logged that `upload_to_lighthouse` was successful, regardless of if it is doing the flipper code or not. IE I would have wanted this here even without this new feature.